### DISCUSSION TITLE
Daily calories in vs calories out bar graph on food logging page for the past 30 days (or flexible days)

### DISCUSSION THREAD

**Discussion by gramapp** [jlai/fitness-dashboard#17] [Ideas]
Feature suggestion,

Daily calories in vs calories out bar graph on food logging tab for the past 30 days (or flexible days)

**Comment by jlai** [comment]
I've added a calories in-vs-out option to the graphs page (currently not adding any graphs or other summaries to the food log page until I can think about the design a bit)

**Reply by chrissard2** [reply]
This looks good.

**Reply by gramapp** [reply]
May be add a legend. Took me a while to figure out that I can hoover a mouse over the bars to know which color is which.
Otherwise thanks, I like it a lot!

**Reply by gramapp** [reply]
Can you also add last 30, 90, 180 days options (and a cherry on top, custom date range). Tracking weight by quarters is not very useful, I think its more useful to see history over X number of last days.

Otherwise, very nice implementation, I like it much better than tine graph in fitbit's dashboard
